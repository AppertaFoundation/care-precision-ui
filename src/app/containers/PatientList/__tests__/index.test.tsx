import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { PatientList } from '../index';

const shallowRenderer = createRenderer();

describe('<PatientList />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(<PatientList />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
