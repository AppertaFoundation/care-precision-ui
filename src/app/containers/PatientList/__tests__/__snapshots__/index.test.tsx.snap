// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PatientList /> should match snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Patient List
    </title>
    <meta
      content="A Patient List"
      name="description"
    />
  </Helmet>
  <Records />
</React.Fragment>
`;
